api/controllers/Depmu_EntryController.js

```diff
@@ -39,7 +39,7 @@ module.exports = {
         _.merge(prebooking, result)),
 
   filterContingencyWithoutCid: prebooking =>
-  !prebooking.contingency || (prebooking.contingency && _.isNull(prebooking.cid_id)),
+  !prebooking.contingency || prebooking.contingency && _.isNull(prebooking.cid_id),
 
```

Comment:
revert this change, as this is suppose to read: 
!prebooking.contingency 
OR
(prebooking.contingency && _.isNull(prebooking.cid_id)),